packages/react-ai/src/components/AIConversation/types.ts

```diff
@@ -75,8 +75,18 @@ export interface SuggestedPrompt {
   inputText: string;
 }
 
+type JSONSchema4TypeName =
```

Comment:
Importing @types/json-schema would introduce a lot of outdated dependencies here, for some reason this wasn't an issue with the previous version release but I believe they changed it

Reply:
Maybe we can give this a nicer reading name?

Reply:
renamed